### PR TITLE
Enotic fixes

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Enotice fixes

Before
----------------------------------------
image

After
----------------------------------------
,mostly gone

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: templates/CRM/Admin/Form/Preferences/Contribute.tpl [hunk 1]
```diff
@@ -7,7 +7,7 @@
  | and copyright information, see https://civicrm.org/licensing       |
  +--------------------------------------------------------------------+
 *}
-<div class="crm-block crm-form-block crm-{$formName}-block">
```
Comment (eileenmcnaughton) [inline templates/CRM/Admin/Form/Preferences/Contribute.tpl:10]: This is never assigned - hence the notice - removing is the status quo

File: templates/CRM/Admin/Form/Preferences/Contribute.tpl [hunk 2]
```diff
@@ -20,7 +20,7 @@
             <td class="label"></td>
             <td>
               {$form.$htmlField.html} {$form.$htmlField.label}
-              {if $desc}
```
Comment (eileenmcnaughton) [inline templates/CRM/Admin/Form/Preferences/Contribute.tpl:23]: yeah never assigned....

**Comment by demeritcowboy** [comment]
str_replace('n', 'r', $title)

File: templates/CRM/Admin/Form/Preferences/Contribute.tpl (hunk 2 above)
Comment (demeritcowboy) [reply templates/CRM/Admin/Form/Preferences/Contribute.tpl:23]: Description now appears!

**Comment by eileenmcnaughton** [comment]
thanks @demeritcowboy

**Pull request merged**